app/views/admin/questions/_question.html.slim

```diff
@@ -8,13 +8,13 @@
         = question.topic.title
     td
       =link_to admin_question_path(question)
-        = truncate(question.content, :length => 50)
+        = truncate(question.content, :length => 70)
     td
       =link_to admin_question_path(question)
-        = truncate(question.answer, :length => 50)
+        = render :partial => "admin/questions/level", :locals => { :@question => question }
```

Comment:
it doesn't matter why you passed it - it's just wrong - "gavnokod" in other words, that's why have code review process where you can use best practices
please read my comment one more time
and do not use instance variable as local